doc/api/stream.md

```diff
@@ -1011,94 +1010,92 @@ However, this would be better implemented as a [Transform][] stream. See
 // alternative example below under the Transform section.
 
 const Readable = require('stream').Readable;
-const util = require('util');
 
-util.inherits(SimpleProtocol, Readable);
+class SimpleProtocol extends Readable {
+  constructor(source, options) {
+    super(options);
+    if (!(this instanceof SimpleProtocol))
+      return new SimpleProtocol(source, options);
```

Comment:
This check is not needed in ES6 classes. The reason this is in the original function version is to ensure that a new instance of  SimpleProtocol (`new SimpleProtocol()`) is returned when SimpleProtocol is called as a function without the `new`. Since ES6 classes require the new, this type of check is no longer necessary.